photon-api/src/integTest/scala/com/linkedin/photon/ml/algorithm/FixedEffectCoordinateTest.scala

```diff
@@ -37,7 +37,7 @@ class FixedEffectCoordinateTest extends SparkTestUtils with GameTestUtils {
 
     // Score before model update
     val score = coordinate.score(model)
-    assertTrue(score.scores.map(_._2).collect.forall(_.score == 0.0))
+    assertTrue(score.scores.map(_._2).collect.forall(scoredDatum => MathUtils.isAlmostZero(scoredDatum.score)))
```

Comment:
I'm gonna leave this one - it's basically identical to this (which I think is what you're asking for)
```
score.scores.map(_._2).collect.foreach(scoredDatum => assertEquals(scoredDatum, 0.0, MathConst.HIGH_PRECISION_THRESHOLD))
```